modules/network-observability-ebpf-agent-alert.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-netobserv-dashboard-ebpf-agent-alerts_{context}"]
+= Using the eBPF agent alert
+
+When the Network Observability eBPF agent hashmap table is full, the eBPF agent processes flows at a degraded performance. When this is the case, an alert shows `NetObservAgentFlowsDropped`. If you see this alert, consider increasing the `cacheMaxFlows` in the `FlowCollector`, as shown in the following example.
+
+.Procedure
+
+. In the web console, navigate to *Operators* → *Installed Operators*.
+
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
```

Comment:
do we use `NetObserv` Operator name in our downstream docs?

Reply:
Good point. I think this is a holdover from previous versions and needs to be updated now.